HHS/Head-Start-TTADP
# [TTAHUB-3451]format comlog comdate history

## Description of change

Improperly formatted `communicationDate` values were making their way into `CommunicationLogs.data`. This fixes the history and produces all valid dates. There's one date in the year 3034, but that's the date that was input by the user.

## How to test

You can see what all the updates will be just by inspecting `SELECT * FROM comdate_corrections;`

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3451


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
